packages/react-composites/src/composites/CallWithChatComposite/components/PreparedDialpad.styles.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IButtonStyles, IModalStyles, Theme } from '@fluentui/react';
+
+import { DialpadStyles } from '@internal/react-components';
+
+/**
+ * @private
+ */
+export const themedDialpadModelStyle = (theme: Theme): Partial<IModalStyles> => ({
+  main: {
+    borderRadius: theme.effects.roundedCorner6,
+    padding: '1rem'
+  }
+});
+
+/**
+ * @private
+ */
+export const themedDialpadStyle = (isMobile: boolean, theme: Theme): Partial<DialpadStyles> => ({
+  root: {
+    padding: 0,
+    marginLeft: 0,
+    marginRight: 0,
+    maxWidth: '100%'
+  },
+  textField: {
+    root: {
+      borderBottom: '1px solid lightgrey'
```

Comment:
Can we use a palette color here? If not what is the reason behind using a static color?